app/controllers/application_controller.rb

```diff
@@ -8,7 +8,7 @@ def ensure_current_user
   end
 
   def current_user
-    @current_user ||= User.find_by(id: session[:user_id])
+    @current_user = User.find_by(id: session[:user_id])
```

Comment:
The `||=` here saves us trips to the database and is often used for methods like `current_user`, please see: http://www.rubyinside.com/what-rubys-double-pipe-or-equals-really-does-5488.html